content/domains/identify/screen-for-substance-disorders/post-assessment.md

```diff
@@ -4,10 +4,21 @@ date: 2019-05-13
 layout: learning-objective-post-assessment
 type: learning-objective-post-assessment
 weight: 3
-q1: ["SENTINAL_0", "SENTINAL_1", "SENTINAL_2", "SENTINAL_3", "SENTINAL_4"]
+q1: ["Strongly disagree", "Disagree", "Neither agree nor disagree", "Agree", "Strongly agree"]
 ---
-Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod
-tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At
-vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren
-no sea takimata sanctus est Lorem ipsum dolor sit amet.
+A 40-year old male patient presents to the clinic in acute pain. While the
+dental assistant begins to take diagnostic radiographs the patient repeatedly
+doses off. When you arrive to the operatory the patient finishes a conversation
+on the telephone then begins to hold his face reporting excruciating pain.
 
+The patient’s wisdom tooth is partially impacted with no caries, no pathology,
+no inflammation of gingiva. As the patient begins to describe his symptoms; you
+notice some of his speech is slurred, and he becomes easily irritated with
+follow-up questions. When you begin your intraoral examination, the patient has
+poor oral hygiene with heavy plaque, heavy calculus build-up and a poor odor.
+You decide to give the patient a prescription for Motrin 600mg and provide him
+with a referral to have the symptomatic tooth extracted at an oral surgeon’s
+office close to your location and schedule a follow-up appointment in 2 weeks.
+
+How comfortable are you addressing with the patient conducting a screening to
```

Comment:
super awkward sentence. (repeated in the pre-assessment.md too). Maybe something like this?

How comfortable are you talking with the patient about a screening to determine risk or presence of substance use disorder?

Reply:
Also...the actual page has this sentence matched with: Strongly agree, etc. Likely this should change to a statement: I am comfortable talking with the patient about a screening to determine risk or presence of substance use disorder.